modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -545,19 +545,56 @@ class NDB_Form_Statistics extends NDB_Form
         } else {
             $Param_Project = '';
         }
-        $subprojects = Utility::getSubprojectsForProject($currentProject);
+        $subprojects[null] = 'All Valid Sub-Projects';
+
+        foreach (Utility::getSubprojectsForProject($currentProject)
+            as $key=>$value) {
+            $subprojects[$key] = $value;
+        }
```

Comment:
I think this could just be:

```
$subprojects = Utility::getSubprojectsForProject($currentProject)
$subprojects[null] = 'All Valid Subprojects'
```
but it's probably not worth taking the time to make the change.